salt/docker/init.sls

```diff
@@ -16,6 +16,7 @@ include:
         Environment="NO_PROXY={{ salt['pillar.get']('proxy:no_proxy', '') }}"
   cmd.run:
     - name: systemctl daemon-reload
+    - stateful: True
```

Comment:
So, from my understanding of the protocol, if `output=None` `=>` _no changes happened_ `=>` _watchers will not be notified_, is that right?